query/src/main/scala/filodb/query/exec/MultiSchemaPartitionsExec.scala

```diff
@@ -39,11 +41,45 @@ final case class MultiSchemaPartitionsExec(queryContext: QueryContext,
   @transient // dont serialize the SelectRawPartitionsExec plan created for plan execution
   var finalPlan: SelectRawPartitionsExec = _
 
+  // Remove _columnName from metricColumn and generate PartLookupResult
+  private def removeColumn(filters: Seq[ColumnFilter], metricName: String,
```

Comment:
This method is doing more than what it's name suggests. It is also doing index lookup. Please fix the surprise by moving the index look up into back into calling method.

Reply:
I am doing index lookup in the method to handle multiple conditions. Renamed method to removeColumnAndGenerateLookupResult

Reply:
sounds good